src/createBreakpoint.ts

```diff
@@ -15,17 +17,14 @@ const createBreakpoint = (
       window.removeEventListener('resize', setSideScreen);
     };
   });
-  const sortedBreakpoints = useMemo(() => Object.entries(breakpoints).sort((a, b) => (a[1] >= b[1] ? 1 : -1)), [
+
+  const sortedBreakpoints = useMemo(() => Object.entries(breakpoints).sort((a, b) => (a[WIDTH] >= b[WIDTH] ? 1 : -1)), [
     breakpoints,
   ]);
-  const result = sortedBreakpoints.reduce((acc, [name, width]) => {
-    if (screen >= width) {
-      return name;
-    } else {
-      return acc;
-    }
-  }, sortedBreakpoints[0][0]);
-  return result;
+
+  const result =
+    sortedBreakpoints.find(([_, width]) => width >= screen) || sortedBreakpoints[sortedBreakpoints.length - 1];
```

Comment:
I guess the main change here is `width >= screen` vs `screen >= width`. What is the advantage of the proposed approach?

This would be a breaking change. I think we can do this change without it being breaking. Possibly,`createBreakpoint` function could receive some parameter that would change its behaviour.

Reply:
huh... it's my fault
`createBreakpoint.ts` This file does not currently exist in the test case.
Can I make this and put it up together?

Reply:
If you could add tests for this file and make it support, both, `>=` and `<` (or possibly user defined function) would be great.